Apeswap-Bots/Large-LP-Deposits-Withdrawals/README.md

```diff
@@ -8,7 +8,7 @@ This bot detects LP deposits and withdrawals in Apeswap pools when the `amount0`
 
 ## Supported Chains
 
-- Binance Smart Chain
+- BNB Smart Chain
```

Comment:
BNB is the token, if you want to use the acronym please use BSC.
- Change to BSC
- or keep Binance Smart Chain as before.

Reply:
I had it the way you suggest initially, but I changed it after reading [this](https://www.binance.com/en/support/announcement/854415cf3d214371a7b60cf01ead0918), can you have a look @e1Ru1o ?

Reply:
Interesting, keep it that way then